Fixed #32539 -- Added toggleable facet filters to ModelAdmin. (#16495)
django/django: docs/ref/contrib/admin/index.txt

    editing, or deleting an object. You can have filters cleared by setting
    this attribute to ``False``.

.. attribute:: ModelAdmin.show_facets

    .. versionadded:: 5.0

    Set ``show_facets`` to ``True`` to have all filter options appended with

>     the count of results that match that filter option in brackets. These

Reviewer: ```suggestion
    the count of results that match that filter option. These
```

It took me a few attempts to understand the 'in brackets' part. Maybe we can drop that part? 🤷‍♂️

Author: Yeah no problem :+1:

    counts will update in line with currently applied filters. By default, this
    is set to ``False``.

>     .. admonition:: Performance considerations with facets

Reviewer: ```suggestion
    .. admonition:: Performance considerations with facets.
```
Does this need a full stop?

Author: A lot of the other admonitions don't have full stops so I guess maybe it's like a title and not needed? I'm not sure though, happy to add it in

    editing, or deleting an object. You can have filters cleared by setting
    this attribute to ``False``.

.. attribute:: ModelAdmin.show_facets

    .. versionadded:: 5.0

>     Controls whether facet counts are displayed for filters in the change-list
>     UI. Default: :attr:`.ShowFacets.ALLOW`.

Reviewer 2: ```suggestion
    Controls whether facet counts are displayed for filters in the admin changelist
    Defaults to :attr:`.ShowFacets.ALLOW`.
```
